core/chains/evm/client/client.go

```diff
@@ -27,7 +27,7 @@ const queryTimeout = 10 * time.Second
 
 // Client is the interface used to interact with an ethereum node.
 type Client interface {
-	cmntypes.Client[*big.Int, common.Address, types.Block, types.Transaction, common.Hash, types.Receipt, types.Log, ethereum.FilterQuery]
+	txmtypes.Client[*big.Int, evmtypes.Nonce, evmtypes.Address, types.Block, types.Transaction, evmtypes.TxHash, types.Receipt, types.Log, ethereum.FilterQuery]
```

Comment:
Didn't you have to implement the method SequenceAt here?

Reply:
Yep, NonceAt gets renamed into SequenceAt